lte/cloud/go/services/lte/obsidian/handlers/handlers.go

```diff
@@ -300,21 +300,35 @@ func listEnodebs(c echo.Context) error {
 	if nerr != nil {
 		return nerr
 	}
+	pageSize, pageToken, err := obsidian.GetPageSizeAndTokenParams(c)
+	if err != nil {
+		return err
+	}
 
-	ents, _, err := configurator.LoadAllEntitiesOfType(
+	ents, nextPageToken, err := configurator.LoadAllEntitiesOfType(
 		nid, lte.CellularEnodebEntityType,
-		configurator.EntityLoadCriteria{LoadMetadata: true, LoadConfig: true, LoadAssocsToThis: true},
+		configurator.EntityLoadCriteria{
+			LoadMetadata:     true,
+			LoadConfig:       true,
+			LoadAssocsToThis: true,
+			PageSize:         uint32(pageSize),
+			PageToken:        pageToken},
```

Comment:
General: move this closing brace to its own line

Reply:
Do we have a team formatter/linter for stuff like this?